docs/manual/guides/tinymce-configuration.de.md

```diff
@@ -0,0 +1,267 @@
+---
+title: "TinyMCE-Editor Konfiguration"
+description: "Die TinyMCE-Editor Konfiguration"
+url: "anleitungen/tinymce-konfiguration"
+weight: 40
+---
+
+
+Zur Bearbeitung der meisten Inhalte wird der [TinyMCE-Editor](https://www.tiny.cloud/) herangezogen. 
+Der Editor ist für Contao bereits vorkonfiguriert. Individuelle Änderungen sind optional. 
+
+
+## Das Template be_tinyMCE.html5
+
+Deine individuelle Konfiguration erfolgt über das Template `be_tinyMCE.html5`. Hier findest du die von Contao gesetzte
+Konfiguration vor und kannst die Angaben updatesicher anpassen.
+
+Erstelle über den [Navigationsbereich](../../administrationsbereich/aufruf-und-aufbau-des-backends/#der-navigationsbereich) 
+»Layout« unter »Templates« ein neues [Template](../../theme-manager/templates-verwalten). 
+Wähle über `Original-Template` das Template `be_tinyMCE.html5` aus und gebe als `Zielverzeichnis` das Hauptverzeichnis an. 
+
+{{% notice note %}}
+Das Template **muss** im Hauptverzeichnis (`templates/be_tinyMCE.html5`) abgelegt werden. Alle Zeilen 
+innerhalb von `<script>...</script>` bis auf die letzte Zeile müssen mit einem **Komma** abgeschlossen werden. 
+Nach dem Speichern des Templates werden deine Änderungen sofort übernommen.
+{{% /notice %}}
+
+
+## Verschiedene Editor Konfigurationen
+
+Wenn du den Template Namen `be_tinyMCE.html5` beibehälst führt dies dazu, dass deine Änderungen sich auf alle Bereiche 
+auswirken die den Editor benutzen. Dies gilt zumindest für die Contao eigenen Komponenten. Mögliche Erweiterungen 
+können u. U. hierbei eigene Templates zur Verfügung stellen.
+
+Du möchstest gezielt eine Editor Konfiguration z. B. nur für das Inhaltselement vom Typ »Text« erstellen? Dazu kannst
+du das Template beliebig umbenennen: z.B. nach `be_myTinyMCE.html5`.
+
+Als nächstes müssen wir Contao dazu bringen dieses Template zu nutzen. Erstelle dir hierzu zwei neue Ordner `contao/dca` 
+im Contao Hauptverzeichnis. Im Verzeichnis `dca` benötigen wir eine neue Datei `tl_content.php` mit folgenden Inhalt:
+
+```php
+// contao/dca/tl_content.php
```

Comment:
Du schreibst, dass wir die Datei mit folgendem Inhalt benötigen:

Dann würde ich <?php mit aufnehmen, was meint ihr?

Reply:
Hm, hatten wir heute auf Slack besprochen. @fritzmg sollte ich das vernachlässigen hinsichtlich der Konsistenz. ?? Bin offen ...

Reply:
Hatten wir auch kurz auf Slack diskutiert. Grundsätzlich verwenden wir (in der Entwicklerdokumentation zumindest) keine öffnenden `<?php` Tags in PHP Beispielen, da das überflüssig ist, weil man das sowieso machen muss. Die DCA Datei kann ja außerdem bereits andere Inhalte haben.

Von daher würde ich das umformulieren, sodass es heißt "… muss folgendes in die die `contao/dca/tl_content.php` eingefügt werden".

Reply:
Ich finde diese Information schadet in der Anwender-Doku nicht.

Reply:
Ich möchte das nicht entscheiden. Eure beide Aussagen sind korrekt ...

Reply:
Dann nimm doch den Vorschlag von fritzmg "… muss folgendes in die die contao/dca/tl_content.php eingefügt werden".

Reply:
OK